q2_vsearch/assets/fastq_stats.html

```diff
@@ -0,0 +1,26 @@
+{% extends "tab-child.html" %}
+
+{% block content %}
+<div class="row">
+    <div class="col-lg-12">
+        <h1>vsearch - fastq_stats</h1>
+        {% set colsize = 6 if paired else 12 %}
+        <div class="col-lg-{{ colsize }}">
+            <h4>Forward Reads</h4>
+            <a href={{ datafiles.forward.stats }}>Download as TXT</a>
+            <p style="margin-top: 15px">
+                <iframe src="{{ datafiles.forward.stats }}" frameborder="0" height="600px" width="95%"></iframe>
```

Comment:
The `iframe` is pretty clunky and I think is a bit difficult to use. How about instead making this a `pre`:

![image](https://user-images.githubusercontent.com/274668/61897690-7ae68580-aecc-11e9-9063-418e6ab3fe7a.png)

Reply:
I looked at it with `pre`, looks nicer, more contrast, better to read.

But what is the clunky part of it for you? Scrolling both files independent from each other? Or both files side by side? In my use case files have different number of lines so I need to go to different heights. Here, I need a scrollbar for each file and want them side by side to make my decisions.

![vsearch01](https://user-images.githubusercontent.com/13518076/61942626-13970700-af9a-11e9-9d45-0a9f5abd96a6.png)